pkg/rewards/claim.go

```diff
@@ -428,6 +474,14 @@ func getEnvFromNetwork(network string) string {
 	}
 }
 
+func stringSplit(toSplit string) []string {
```

Comment:
this method is redundant, why not just use `strings.split(array)` it always returns a `[]string`

Reply:
The test case I added is related to this. If a value of "" is passed to `strings.split`, the result is a slice of length 1 storing the empty string. When this is passed to `stringToAddressArray`, we get the `HexToAddress` value of the empty string, which is the 0x000.. address.

Reply:
I see - but that will anyway get filtered away in your filteration logic since they won't be in claimable tokens. So I think this is not required. If you still want it, I would rather explicitly filter out Zero Address since if someone reads this code at later point it time, they know what is happening.

Reply:
Actually now that I read the code again, my comment won't. 

But I still think that we would rather explicitly filter out ZERO address then have a custom string split logic. It's not implicit that custom split is done to remove zero address.

Reply:
discussed offline and agreed on approach.